src/metric-engine/src/engine/alter.rs

```diff
@@ -150,10 +150,16 @@ impl MetricEngineInner {
         region_id: RegionId,
         request: RegionAlterRequest,
     ) -> Result<()> {
-        info!("Metric region received alter request {request:?} on physical region {region_id:?}");
-        FORBIDDEN_OPERATION_COUNT.inc();
-
-        ForbiddenPhysicalAlterSnafu.fail()
+        match request.kind {
+            AlterKind::SetRegionOptions { options: _ }
+            | AlterKind::UnsetRegionOptions { keys: _ } => Ok(()),
+            _ => {
+                info!("Metric region received alter request {request:?} on physical region {region_id:?}");
+                FORBIDDEN_OPERATION_COUNT.inc();
+
+                ForbiddenPhysicalAlterSnafu.fail()
+            }
+        }
```

Comment:
You need to alter the underlying mito region as well. i.e., `self.mito.handle_request(region_id, request)`

Reply:
Thank you for catching this. I have two follow up questions:
- is there a place that i can add a test for this senario?
- I see `alter_logical_region` only support add column request, does that mean we can only set ttls to physical table on metric engine?

Reply:
> is there a place that i can add a test for this scenario?

You can look at a test named `test_alter_column_fulltext_options` in the `mito` crate.

> does that mean we can only set ttls to physical table on metric engine?

Yes, we can set TTLs and other table options of the physical table.